examples/azure-native-sample-project/README.md

```diff
@@ -0,0 +1,30 @@
+# Azure Native sample project
+
+This project creates a VM instance in Azure Cloud using the Pulumi Kotlin Azure native SDK.
+
+## How to run
+
+1. Install Pulumi (see: [docs](https://www.pulumi.com/docs/install/)).
+2. Log in to your Pulumi account (see: [docs](https://www.pulumi.com/docs/cli/commands/pulumi_login/)).
+   ```bash
+   pulumi login
+   ```
+3. Install the Azure CLI (see: [docs](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli)).
+4. Authenticate to Azure (see:
+   [docs](https://www.pulumi.com/registry/packages/azure-native/installation-configuration/#authentication-methods)).
+   ```bash
+   az login
+   ```
+5. Set the necessary Pulumi configuration properties.
+   ```bash
+   pulumi config set azure-native:location westeurope 
+   ```
+6. Create a stack and perform the update.
+   ```bash
+   pulumi up
+   ```
+7. Don't forget to delete the created resources and stack.
+   ```bash
+   pulumi down
```

Comment:
It's `destroy` not `down` 😁

Reply:
I think `down` also works! Or at least it looks like I've been using it according to my terminal history. But true, it's supposed to be `destroy` :)

Reply:
You are right! Interesting, didn't know that one :).

```
> pulumi down --help

... some stuff ...

Usage:
  pulumi destroy [flags]

Aliases:
  destroy, down

... some stuff ...
```

It's good that you've changed it `destroy`, though. It minimizes the probability of surprise :).